<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>multibootusb</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../multibootusb_old release/tools/multibootusb.png</normaloff>../../../multibootusb_old release/tools/multibootusb.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>MultiBootUSB</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="6" column="0" colspan="5">
          <widget class="QLabel" name="status">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QPushButton" name="create">
           <property name="text">
            <string>Create</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item row="4" column="4">
          <widget class="QLabel" name="labelstep2">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Step 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="2">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Uninstall (Optional)</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QPushButton" name="uninstall">
                <property name="text">
                 <string>Uninstall Distro</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QPushButton" name="browse_iso">
           <property name="text">
            <string>Browse ISO</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="4">
          <widget class="QListWidget" name="listWidget"/>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>USB Details</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="usb_dev">
              <property name="text">
               <string>Drive ::</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="usb_label">
              <property name="text">
               <string>Label ::</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="usb_mount">
              <property name="text">
               <string>Mount::</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="usb_size_ttl">
              <property name="text">
               <string>Size Total ::</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="usb_size_avl">
              <property name="text">
               <string>Size Avail ::</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QLabel" name="labelstep3">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Step 3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="5">
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QPushButton" name="close">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="1" column="2" colspan="2">
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QGroupBox" name="groupBox_11">
             <property name="title">
              <string>Detect USB</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QPushButton" name="detect_usb">
                <property name="text">
                 <string>Referesh USB</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="labelstep1">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Step 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="syslinux_ab">
      <attribute name="title">
       <string>Syslinux</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Install Syslinux</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="2" column="1">
               <widget class="QPushButton" name="install_syslinux">
                <property name="text">
                 <string>Install</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QRadioButton" name="install_sys_all">
                <property name="text">
                 <string>Install syslinux and copy all required files.</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QRadioButton" name="install_sys_only">
                <property name="text">
                 <string>Install only syslinux (existing configurations will not be altred).</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Edit syslinux.cfg</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="1" column="1">
               <widget class="QPushButton" name="edit_syslinux">
                <property name="text">
                 <string>Edit</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Using this option user can edit syslinux.cfg file directly. It directly uses &lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;default editor of host system. Be careful while editing syslinux.cfg file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>QEMU</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Boot ISO</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="4" column="4">
             <widget class="QRadioButton" name="ram_iso_2048">
              <property name="text">
               <string>2048 MB</string>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QRadioButton" name="ram_iso_1024">
              <property name="text">
               <string>1024 MB</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QRadioButton" name="ram_iso_256">
              <property name="text">
               <string>256 MB</string>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QPushButton" name="browse_iso_qemu">
              <property name="text">
               <string>Browse ISO</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="5">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Best way to check downloaded iso. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QRadioButton" name="ram_iso_512">
              <property name="text">
               <string>512 MB</string>
              </property>
             </widget>
            </item>
            <item row="6" column="4">
             <widget class="QPushButton" name="boot_iso_qemu">
              <property name="text">
               <string>Boot ISO</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QRadioButton" name="ram_iso_768">
              <property name="text">
               <string>768 MB</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="4">
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
            <item row="3" column="1">
             <spacer name="verticalSpacer_8">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="6" column="0" colspan="4">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Choose desired RAM and click on Boot ISO button.</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <spacer name="verticalSpacer_7">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <spacer name="verticalSpacer_9">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Boot USB</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="2" column="2">
               <widget class="QRadioButton" name="ram_usb_768">
                <property name="text">
                 <string>768 MB</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QRadioButton" name="ram_usb_256">
                <property name="text">
                 <string>256 MB</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="5">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Use this option if you want to check USB installation.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QRadioButton" name="ram_usb_1024">
                <property name="text">
                 <string>1024 MB</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="4">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Choose desired RAM and click on Boot USB button.</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QRadioButton" name="ram_usb_512">
                <property name="text">
                 <string>512 MB</string>
                </property>
               </widget>
              </item>
              <item row="4" column="4">
               <widget class="QPushButton" name="boot_usb_qemu">
                <property name="text">
                 <string>Boot USB</string>
                </property>
               </widget>
              </item>
              <item row="2" column="4">
               <widget class="QRadioButton" name="ram_usb_2048">
                <property name="text">
                 <string>2048 MB</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="2">
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="5" column="0">
               <spacer name="verticalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="settings">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>Persistance</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox_9">
                <property name="title">
                 <string>Ubuntu</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QCheckBox" name="checkBox">
                   <property name="text">
                    <string>Enable persistance</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable if you would like &lt;/p&gt;&lt;p&gt;to have persistancefor &lt;/p&gt;&lt;p&gt;ubuntu and its derivatives.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QGroupBox" name="groupBox_10">
                <property name="title">
                 <string>Debian</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <widget class="QCheckBox" name="checkBox_2">
                   <property name="text">
                    <string>Enable persistance</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable if you would like &lt;/p&gt;&lt;p&gt;to have persistancefor &lt;/p&gt;&lt;p&gt;debian and its derivatives.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string>Update</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="0" column="0">
               <widget class="QCheckBox" name="checkBox_3">
                <property name="text">
                 <string>Check update</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable this option if you would like to recieve update information &lt;/p&gt;&lt;p&gt;right in to your desktop.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="1">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>30</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;An advanced bootable usb creator with option to install/uninstall &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;multiple distros. This software is written in python and pyqt. &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Copyright 2010-2014 Sundar&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Author(s)&lt;/span&gt;: Sundar, Ian Bruce&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Licence:&lt;/span&gt; GPL version 2 or later&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Home page&lt;/span&gt;: &lt;a href=&quot; http://multibootusb.org&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;http://multibootusb.org&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Help/Email:&lt;/span&gt; feedback.multibootusb@gmail.com&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Source Code: &lt;/span&gt;&lt;a href=&quot;https://github.com/mbusb/multibootusb&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://github.com/mbusb/multibootusb&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
